### ISSUE TITLE
Add some way to create release from the version in a package.json

### ISSUE THREAD

**Issue by heff** [heff/chg#1]
Right now there's no way to just read the package.json file and use what's in there. Coudl provide some way to say: chg release package.json, or something like it.

**Comment by Hypercubed** [comment]
Yes, would ne nice if you could just do `npm version [major|minor|patch]` and have chg update CHANGELOG as expected.  Are you still working on this?

**Comment by mmcc** [comment]
We haven't done any work towards this specifically since we didn't go this route for Video.js. I still think this is really relevant, though, so any interest in putting together a PR for this?

**Comment by Hypercubed** [comment]
I added a `-y` flag to the release command, since `chg release` already reads `package.json` when no version is given.

By "we didn't go this route" does that mean you used something else for Video.js?

**Comment by mmcc** [comment]
Nah, I just meant we handled it differently with VJS, but we're still using chg.

**Comment by gkatsev** [comment]
I'm not sure that it's possible to automatically bind into `npm version` but npm's scripts allow you to add `pre` and `post` `version` scripts. So, you could add chg into a `postversion` step to have it be done automatically. Or maybe even `preversion`.

**Comment by Hypercubed** [comment]
`preversion` is before the version number is bumped in package.json.  `version` is after the bump.  See https://docs.npmjs.com/cli/version .
